testbed/testbed/senders.go

```diff
@@ -68,6 +69,7 @@ type MetricDataSender interface {
 // DataSenderOverTraceExporter partially implements TraceDataSender via a TraceExporter.
 type DataSenderOverTraceExporter struct {
 	exporter exporter.TraceExporter
+	Address  string
```

Comment:
Is this used anywhere?

Reply:
No, sorry I was thinking to use it instead of `Port` and later I forgot to delete it.